docker/wordpress/wp-content/plugins/a/a.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
Do we want to ship the toy plugin with Jetpack? It's mostly meant to test things, right? I don't know if we need to include it with every dev environment from now on?

Reply:
I'd expect we don't ship this, but keep it somewhere as a valid PoC for any other plugins that would want to consume our packages.

Perhaps it makes sense to add the toy plugin as an example in the documentation?

Reply:
yeah, it was meant to test this PR, we can remove it once it is done, or I can move it somewhere else we could cherry pick to quickly add plugins to our testing instructions.